robots/quarantine/cmd/types.go

```
@@ -101,4 +102,16 @@ type MostFlakyTestsTemplateData struct {
 	MostFlakyTestsBySig map[string]TestsPerSIG
 	SIGs                []string
 	TestNames           []string
+	RequiredJobs        map[string]struct{}
+}
+
+// IsOptionalLane reports whether a search.ci job name is not a required presubmit.
+// Jobs missing from the required set (optional, skip_report, release-branch suffixes,
+// or otherwise unknown) are tagged optional. 
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)